### PR TITLE
[PATCH][EDA-1734] - Change settings for database in Django

### PR DESCRIPTION
**Pull request by npaura-eb**
Inside the configurations of the database, in the attribute name, change the key `dbClusterIdentifier` for `dbname`.

### CODE REVIEW

File: edagames/settings/prod.py [hunk 1]
```diff
@@ -128,7 +128,7 @@ def get_secret(db_secret_name):
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.mysql',
-        'NAME': secret_value["dbClusterIdentifier"],
+        'NAME': secret_value["dbname"],
```
Comment (mmerida-eb) [inline edagames/settings/prod.py:131]: Why is this change necessary?

Comment (npaura-eb) [reply edagames/settings/prod.py:131]: Because the `dbClusterIdentifier` key of the secret value of AWS Secrets Manager is the name of the cluster, in our case,`aurora-edagames`. And Django needs the name of the database, not the cluster. So, I have to add a new key in the secret, called `dbname` with the name of the database as its value.

**Review by mmerida-eb — APPROVED** [review]
LGTM
